generator/generator.py

```diff
@@ -9,7 +9,16 @@
 from struct import pack
 import base64
 
-from generator_lib import generators
+# Generator functions
+# Add your new generator function here
+# TODO: with some import syntax or clever moduling, it might be possible not
+# to have to do anything here?
```

Comment:
Yeah, that would be pretty cool! Did you raise an issue for this, btw, at least to investigate? I don't think it's very clear what you mean just from this TODO

Reply:
Yep, will raise an issue after this.